Plugins~/Src/MeshSyncClientBlender/python/3.1.0/unity_mesh_sync.py

```diff
@@ -340,26 +339,29 @@ def draw(self, context):
 
 # ---------------------------------------------------------------------------------------------------------------------
 
-classes = (
+from .unity_mesh_sync_baking import MESHSYNC_PT_Baking
+
+classes = [
     MESHSYNC_PT_Main,
     MESHSYNC_PT_Server,
     MESHSYNC_PT_Scene,
     MESHSYNC_PT_UnityProject,
     MESHSYNC_PT_Animation,
+    MESHSYNC_PT_Baking,
     MESHSYNC_PT_Cache,
     MESHSYNC_PT_Version,
     MESHSYNC_OT_SendObjects,
     MESHSYNC_OT_SendAnimations,
     MESHSYNC_OT_AutoSync,
     MESHSYNC_OT_ExportCache,
-    MESHSYNC_Preferences,
-)
+    MESHSYNC_Preferences
+] + sharedClasses
 
 def register():
-    msb_initialize_properties()
     bpy.app.handlers.load_post.append(MESHSYNC_OT_AutoSync.load_handler)
     for c in classes:
         bpy.utils.register_class(c)
+    msb_initialize_properties()
```

Comment:
We are now using a property group and a collection properties of the new group, which requires the class for it to be registered before it can be used.